common/effects.cpp

```diff
@@ -604,16 +604,8 @@ bool is_building_replaced(const struct city *pcity,
 }
 
 /**
-   Returns the effect bonus of a given type for any target.
-
-   target gives the type of the target
-   (player,city,building,tile) give the exact target
-   effect_type gives the effect type to be considered
-
-   Returns the effect sources of this type _currently active_.
-
-   The returned vector must be freed (building_vector_free) when the caller
-   is done with it.
+ * This function is deprecated. If you want to add new parameters, switch to
+ * using get_target_bonus or get_target_bonus_effects by req_context.
  */
 int get_target_bonus_effects(
```

Comment:
Let's remind ourselves to remove this and other deprecated functions. In the headers:
```cpp
[[deprecated]]
int get_target_bonus_effects(...)
```